src/stories/components/Groups/Code Examples.mdx

```diff
@@ -0,0 +1,76 @@
+import { Meta, Canvas, Markdown } from '@storybook/blocks';
+import * as GroupsStories from './Groups.stories';
+import UseGroupsReturn from '../../hooks/UseGroups/UseGroupsReturn.md?raw';
+/* This "?raw" import raw text form the md file */
+
+<Meta of={GroupsStories} />
+
+# `Groups`
+
+A separate filter component that allows you to filter down by groups.
+
+> Note: In Constructor, groups are typically categories and are created via the `item_groups.csv` feed.
+
+**Dynamic Hierarchical Options Rendering:**
+
+- Automatically updates and displays selected filters based on user requests
+- Displays the current selected group hierarchy, and allows for backward navigation
```

Comment:
I agree but also I disagree. I think the Breadcrumbs should have this description since it's meant for use as a top-level component in the BrowseGrid, but I think it's fair that the Groups Component also has it could exist on it's own in the SearchGrid. The thought process is "If I include this component, what features am I including?"